src/components/Video/Video.jsx

```diff
@@ -7,9 +7,45 @@ import { useFilter } from "../../context/filter-context";
 
 function Video({ video }) {
   const { productDispatch } = useFilter();
-  const { isLiked } = video;
   var comments = video.comments
   const [modal, setModal] = useState(false)
+
+  const [like, setlike] = useState(128);
+  const [dislike, setdislike] = useState(6);
+
+  const [likeactive, setlikeactive] = useState(false);
+  const [dislikeactive, setdislikeactive] = useState(false);
+
+  function likef() {
+    if (likeactive) {
+      setlikeactive(false);
+      setlike(like - 1);
+    } else {
+      setlikeactive(true);
+      setlike(like + 1);
+      if (dislikeactive) {
+        setdislikeactive(false);
+        setlike(like + 1);
+        setdislike(dislike - 1);
+      }
+    }
+  }
+
+  function dislikef() {
+    if (dislikeactive) {
+      setdislikeactive(false);
+      setdislike(dislike - 1);
+    } else {
+      setdislikeactive(true);
+      setdislike(dislike + 1);
+      if (likeactive) {
+        setlikeactive(false);
+        setdislike(dislike + 1);
+        setlike(like - 1);
+      }
+    }
+  }
+
```

Comment:
you can use a reducer rather than using multiple useStates, this will also abstract out all this code

Reply:
sure, will keep this in mind from next time